scripts/windows/generate-windows-version-info/main.go

```diff
@@ -22,16 +29,16 @@ func main() {
 	flag.Parse()
 
 	if version == "" {
-		fatalf("version is required")
+		err = fmt.Errorf("%s", "version is required")
```

Comment:
I'm afraid this approach does not stop execution as `fatalf` did. This will result in unpredictable failures.

Instead, we could wrap these steps with another function, and handle errors as usual:

```go
func main() {
	var version string
	flag.StringVar(&version, "version", "", "Version in semver format.")

	var outputPath string
	flag.StringVar(&outputPath, "out", "", "Output file path.")

	var iconPath string
	flag.StringVar(&iconPath, "icon", "", "Icon file path.")
	flag.Parse()

	if err := generateVersionInfo(version, outputPath, iconPath); err != nil {
		_, _ = fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
}

func generateVersionInfo(version, outputPath, iconPath string) error {
 	version = strings.Trim(version, `"`)
 	v, err := semver.Parse(strings.TrimPrefix(version, "v"))
 	if err != nil {
 		return fmt.Errorf("invalid version %q: %w", version, err)
 	}
 	if outputPath == "" {
 		return fmt.Errorf("output path is required")
 	}
 	// ... all remaining steps.
 	return nil
}
```